src/app/modules/players/players.controller.js

```diff
@@ -148,9 +149,21 @@ angular
        */
       function startMatch() {
         Match.reset();
+        localStorage.setItem('controlPlayer','0');
```

Comment:
A cosa serve e cosa vuoi salvare?

Reply:
Ho usato "controlPlayer" come appoggio per la funzione "currentPlayer()" nel file sumray.controller.js 
Quando voglio iniziare una nuova partita, premendo il tasto "start" setto "controlPlayer" a 0 in modo tale che quando viene richiamata la funzione "currentPlayer()" posso entrare nell' if e quindi aggiornare il nome del giocatore.
Al contrario, la funzione "loadMatch()" me lo setta a 1 così da poter riprendere quello che avevo in precedenza salvato in "currentPlayer".